components/fatfs/src/ffconf.h

```diff
@@ -1,3 +1,6 @@
+#ifndef _FFCONF_DEFINED
```

Comment:
ESP-IDF uses `#pragma once` guards, instead of the old `#ifndef` ones

Reply:
I looked in the neighboring files of this component and did same

Reply:
Fair enough.
It is a bit of an edge case, since FATFS is basically an "external component".